### PR TITLE
Add ConvertFrom-SecureString -AsPlainText documentation

### PR DESCRIPTION
**Pull request by rjmholt**
# PR Summary
Documents the new `-AsPlainText` parameter on `ConvertFrom-SecureString`.

## PR Context

`-AsPlainText` was added to `ConvertFrom-SecureString` in https://github.com/PowerShell/PowerShell/pull/11142.

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [x] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md [hunk 1]
```diff
@@ -16,13 +17,16 @@ Converts a secure string to an encrypted standard string.
...
 
 ### Secure (Default)
-
 ```
 ConvertFrom-SecureString [-SecureString] <SecureString> [[-SecureKey] <SecureString>] [<CommonParameters>]
 ```
 
-### Open
+### AsPlainText
+```
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md]: New line under headers

File: reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md [hunk 2]
```diff
@@ -85,8 +89,32 @@ Because each decimal numeral represents a single byte (8 bits), the key has 24 d
 The second command uses the key in the `$Key` variable to convert the secure string to an encrypted
 standard string.
 
+### Example 4: Convert a secure string directly to a plaintext string
+
+```
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md]: Should be tagged 'powershell'

File: reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md [hunk 3]
```diff
@@ -85,8 +89,32 @@ Because each decimal numeral represents a single byte (8 bits), the key has 24 d
...
+$secureString = ConvertTo-SecureString -String 'Example' -AsPlainText
+$secureString # 'System.Security.SecureString'
+ConvertFrom-SecureString -SecureString $secureString -AsPlainText # 'Example'
+```
+
 ## PARAMETERS
 
+### -AsPlainText
+When set, **ConvertFrom-SecureString** will convert secure strings to the decrypted plaintext string
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md]: Should be code fenced `ConvertFrom-SecureString`

File: reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md [hunk 4]
```diff
@@ -137,10 +165,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable,
--InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose,
--WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](../Microsoft.PowerShell.Core/About/about_CommonParameters.md).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (chasewilson) [inline reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md]: Should be:

This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).

File: reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md [hunk 5]
```diff
@@ -85,8 +89,32 @@ Because each decimal numeral represents a single byte (8 bits), the key has 24 d
...
+```powershell
+$secureString = ConvertTo-SecureString -String 'Example' -AsPlainText
+$secureString # 'System.Security.SecureString'
+ConvertFrom-SecureString -SecureString $secureString -AsPlainText # 'Example'
+```
+
 ## PARAMETERS
 
+### -AsPlainText
+When set, `ConvertFrom-SecureString** will convert secure strings to the decrypted plaintext string
```
Comment (sdwheeler) [inline reference/7.0/Microsoft.PowerShell.Security/ConvertFrom-SecureString.md]:
```suggestion
When set, `ConvertFrom-SecureString` will convert secure strings to the decrypted plaintext string
```

**Pull request merged**